core/appeal/service.go

```diff
@@ -188,25 +188,54 @@ func (s *Service) Create(ctx context.Context, appeals []*domain.Appeal, opts ...
 		resourceIDs = append(resourceIDs, a.ResourceID)
 		accountIDs = append(accountIDs, a.AccountID)
 	}
-	resources, err := s.getResourcesMap(ctx, resourceIDs)
-	if err != nil {
-		return err
-	}
-	providers, err := s.getProvidersMap(ctx)
-	if err != nil {
-		return err
-	}
-	policies, err := s.getPoliciesMap(ctx)
-	if err != nil {
-		return err
-	}
 
-	pendingAppeals, err := s.getAppealsMap(ctx, &domain.ListAppealsFilter{
-		Statuses:   []string{domain.AppealStatusPending},
-		AccountIDs: accountIDs,
+	eg, egctx := errgroup.WithContext(ctx)
+	var resources map[string]*domain.Resource
+	var providers map[string]map[string]*domain.Provider
+	var policies map[string]map[uint]*domain.Policy
+	var pendingAppeals map[string]map[string]map[string]*domain.Appeal
```

Comment:
we could club into this
```suggestion
	var (
	    resources map[string]*domain.Resource
	    providers map[string]map[string]*domain.Provider
	    policies map[string]map[uint]*domain.Policy
	    pendingAppeals map[string]map[string]map[string]*domain.Appeal
	)
```